designer/client/cypress/e2e/suggester.cy.ts

```diff
@@ -1,41 +1,46 @@
 describe("Expression suggester", () => {
-  const seed = "suggester"
+    const seed = "suggester";
 
-  before(() => {
-    cy.deleteAllTestProcesses({filter: seed, force: true})
-  })
+    before(() => {
+        cy.deleteAllTestProcesses({ filter: seed, force: true });
+    });
 
-  after(() => {
-    cy.deleteAllTestProcesses({filter: seed})
-  })
+    after(() => {
+        cy.deleteAllTestProcesses({ filter: seed });
+    });
 
-  it("should display colorfull and sorted completions", () => {
-    cy.visitNewProcess(seed, "variables")
-    cy.layoutScenario()
-    cy.get("[model-id=kafka-string]").trigger("dblclick")
-    cy.get("[data-testid=window]").as("modal")
-    cy.get("[title=Value]").next().find(".ace_editor").click().type(".").contains(/\.$/)
-    cy.get(".ace_autocomplete").should("be.visible").matchImage({
-      maxDiffThreshold: 0.0025,
-      screenshotConfig: {padding: [25, 1, 1]},
-    })
-    cy.get("[title=Value]").next().find(".ace_editor").click().type("c").contains(/\.c$/)
-    cy.get(".ace_autocomplete").should("be.visible").matchImage({
-      maxDiffThreshold: 0.0025,
-      screenshotConfig: {padding: [25, 1, 1]},
-    })
-  })
+    it("should display colorfull and sorted completions", () => {
+        cy.visitNewProcess(seed, "variables");
+        cy.layoutScenario();
+        cy.get("[model-id=kafka-string]").trigger("dblclick");
+        cy.get("[data-testid=window]").as("modal");
+        cy.get("[title=Value]").next().find(".ace_editor").click().type(".").contains(/\.$/);
+        cy.get(".ace_autocomplete")
+            .should("be.visible")
+            .matchImage({
+                maxDiffThreshold: 0.0025,
+                screenshotConfig: { padding: [25, 1, 1] },
+            });
+        cy.get("[title=Value]").next().find(".ace_editor").click().type("c").contains(/\.c$/);
+        cy.get(".ace_autocomplete")
+            .should("be.visible")
+            .matchImage({
+                maxDiffThreshold: 0.0025,
+                screenshotConfig: { padding: [25, 1, 1] },
+            });
+    });
 
-  it("should display completions for second line (bugfix)", () => {
-    cy.visitNewProcess(seed, "variables")
-    cy.layoutScenario()
-    cy.get("[model-id=kafka-string]").trigger("dblclick")
-    cy.get("[data-testid=window]").as("modal")
-    cy.get("[title=Value]").next().find(".ace_editor").click().type("{enter}#").contains(/^#$/m)
-    cy.get(".ace_autocomplete").should("be.visible")
-      .matchImage({
-        maxDiffThreshold: 0.0025,
-        screenshotConfig: {padding: [45, 1, 1]},
-      })
-  })
-})
+    it("should display completions for second line (bugfix)", () => {
+        cy.visitNewProcess(seed, "variables");
+        cy.layoutScenario();
+        cy.get("[model-id=kafka-string]").trigger("dblclick");
+        cy.get("[data-testid=window]").as("modal");
+        cy.get("[title=Value]").next().find(".ace_editor").click().type("{enter}.").contains(/^.$/m);
```

Comment:
We can't just replace `#` with `.`. Test name is now no relevant. Let's change the expression to work with suggestions mechanism. Maybe:
```
#mapVarname.camelCase +
#<suggestions here>
```
WDYT?

Reply:
Good idea